### ISSUE TITLE
Blank window for Nativefied bet365

### ISSUE THREAD

**Issue by sungnix** [nativefier/nativefier#1236]
**Description**
I have nativefied a bunch of sites (Google, Twitter, etc.).
But bet365 gives me a blank screen on opening.

**Steps to reproduce issue**
nativefier --name "bet" "https://www.bet365.com"

**Details**
Full nativefier command used to build your app: nativefier --name "bet" "https://www.bet365.com"
Version of nativefier : v42.2.1
Version of node.js : v15.10.0
OS: macOS 11.4
Model: MacBook Air (M1, 2020)

**Comment by TheCleric** [comment]
I can reproduce this issue, but do not think we can fix it. It looks like that site has some HEAVLY obfuscated JavaScript that is causing an infinite loop in the Electron BrowserWindow.

For reference see: https://github.com/electron/electron/issues/27577

Since that was a slightly different domain, I did look at the source for https://www.bet365.com/ and did find an extremely similar piece of javascript to what came from bet365.it in that ticket, so I feel fairly confident in this.

So long story short, unfortunately it's out of our hands. Sorry.
